static/app/utils/discover/charts.tsx

```diff
@@ -92,6 +92,8 @@ export function axisLabelFormatterUsingAggregateOutputType(
       return axisDuration(value, durationUnit);
     case 'size':
       return formatBytesBase2(value, 0);
+    case 'rate':
+      return `${value}/s`;
```

Comment:
Oohh, risky! I think this should accept a `rateUnit`, there have been some discussions that per second might not make sense for people with lower throughput. Is it a lot of work to do that?

Reply:
You're right, I thought about this but thought it wasn't necessary. I can certainly look into making it dynamic based on whether it's higher or lower throughput, but maybe we should wait for confirmation on that before I work on it?

Reply:
Take a look, IMO! Fingers crossed it's easy, and if yes, it'll be so worth it. If not, then no worries. If it was just Starfish code I wouldn't even ask, but this is common Discover stuff, and it deserves to be solid

Reply:
Fair point, lemme go do that then